Themes/default/css/index.css

```diff
@@ -481,6 +481,59 @@ blockquote cite::before {
 .bbc_h6 {
 	font-size: 1em;
 }
+.bbc_details,
+.bbc_summary,
+.bbc_details_content {
+	margin: 5px;
+}
+.bbc_details {
+	border: 1px dotted #aaa;
+	margin: 5px;
+}
+.bbc_summary {
+	cursor: pointer;
+	margin: 5px;
+}
+.bbc_details[open] > .bbc_summary {
+    padding-bottom: 7px;
+	border-bottom: 1px dotted #aaa;
+}
+.bbc_inline_spoiler {
+	cursor: pointer;
+	border-radius: 3px;
+	transition: background 0.5s;
+	background-color: currentcolor;
+	position: relative;
+}
+.bbc_inline_spoiler.revealed {
+	background-color: transparent;
+}
+.bbc_inline_spoiler button {
```

Comment:
To be on the safe side, since this BBCode can include other tags, should this be `.bbc_inline_spoiler > button` (and similar in line 522, 525 & 534)?

Reply:
Yes, that is a good idea.